inst/stan/functions/expected_obs_from.stan

```diff
@@ -143,7 +141,7 @@ vector expected_obs_from_snaps(int start, int end, array[] vector imp_obs,
       esnap += l;
       log_exp_obs[ssnap:esnap] = expected_obs_from_index(
         i, imp_obs, rdlurd, srdlh, refp_lh, dpmfs, ref_p, rep_h, ref_as_p, g, t,
-        l, refnp_lh, ref_np, p
+        l, refnp_lh, ref_np, p, agg_probs, agg_indicators
```

Comment:
See the index lookuip on 133-136 for a sense of the information we have to play with here.